hicommonwealth/commonwealth
# [GATING] - Create and Edit Group UI

## Link to Issue
Closes: 
https://github.com/hicommonwealth/commonwealth/issues/4991
https://github.com/hicommonwealth/commonwealth/issues/5150

## Description of Changes
- Added "create group" and "update group" page UI's and forms with dynamic validation and error states. Accessible at `/:scope/members/groups/create` and `/:scope/members/groups/:id/update`.
- Only admins of a community can access these pages
- Connected mock API (TODO: Connect the real API once developed)
- Added new components, Select List from https://github.com/hicommonwealth/commonwealth/issues/5085 and a fused-in Radio label with text input (its not generic)

## "How We Fixed It"
This is a new feature.

## Test Plan
Since this is a new feature

- Visit the pages (`/:scope/members/groups/create` and `/:scope/members/groups/:id/update`) and confirm the page UI matches the expected UI defined in https://github.com/hicommonwealth/commonwealth/issues/4991
and https://github.com/hicommonwealth/commonwealth/issues/5150.
- Visit the pages (`/:scope/members/groups/create` and `/:scope/members/groups/:id/update`) and confirm the product copy/ux matches the expected copy/ux defined in https://github.com/hicommonwealth/commonwealth/issues/4991
and https://github.com/hicommonwealth/commonwealth/issues/5150.
- Visit the pages (`/:scope/members/groups/create` and `/:scope/members/groups/:id/update`) and confirm the engineering behavior matches the expected behavior defined in https://github.com/hicommonwealth/commonwealth/issues/4991
and https://github.com/hicommonwealth/commonwealth/issues/5150.

## Deployment Plan
N/A

## Other Considerations
TODO: Connect the real API once developed
